server/build.gradle

```diff
@@ -111,6 +111,12 @@ dependencies {
   api "org.apache.lucene:lucene-spatial3d:${versions.lucene}"
   api "org.apache.lucene:lucene-suggest:${versions.lucene}"
 
+  api 'org.apache.shiro:shiro-core:1.9.1'
+  api 'org.slf4j:slf4j-api:1.7.36'
```

Comment:
This is clashing with the slf4j version in `buildSrc/version.properties` and causing problems when running `./gradlew clean assemble`. This version is newer than the one in `version.properties`, perhaps we should consider bumping the version in `version.properties`.